src/runtime_src/doc/toc/xrt_native_apis.rst

```diff
@@ -800,6 +800,68 @@ The above code shows
 
 
 
+Event Profile
+-------------
+
+In Versal ACAPs with AI Engines, the XRT Event class (``xrt::aie::event``) and its member functions can be used to configure AI Engine hardware resources for performance profiling and event tracing.
+
+Create Event
+~~~~~~~~~~~~
+
+The class constructor ``xrt::aie::event`` is used to create event object as shown below 
+
+.. code:: c
+      :number-lines: 35
+           
+           auto event = xrt::aie::event(device);
+           
+
+The event object can be used to execute the profiling functions and collect profile statistics by calling event APIs.
+
+Start Profiling
+~~~~~~~~~~~~~~~
+
+The member function ``xrt::aie::event::start_profiling()`` is used to start performance counters in AI Engine as per the profiling option passed as an argument. This function configures the performance counters in the AI Engine and starts profiling.
+
+
+.. code:: c
+      :number-lines: 45
+           
+           auto graph = xrt::graph(device, xclbin_uuid, "graph_name");
+           auto handle = event.start_profiling(int option, std::string& port1, std::string& port2, int value);
```

Comment:
I was hoping this documentation would explain the semantics of option and value :-(
```suggestion
           auto handle = event.start_profiling(option, std::string& port1, std::string& port2, int value);
```